docs/pywbemcli/commands.rst

```diff
@@ -1559,6 +1563,77 @@ Example:
+WBEM server.
+
+The overview reports a summary by namespace of this information and the
+detail view (``server schema --detail`) reports the information for each
+:ref:`Schema` in the namespace.
+
+For each CIM schema in each namespace, it provides information on the
+:ref:`Schema` release version  whether the :ref:`Schema` has experimental
+classes, and the number of classes in the :ref:`Schema`.
+
+Pywbemcli determines the version of the :ref:`Schema` by finding the highest
+value of the Version qualifier on any of the classes in the namespace that
+are in that :ref:`Schema`
+
+For the :ref:`DMTF schema` in the overview report the highest value is
+determined by finding the highest version value for any of the DMTF schemas
+(normally 'CIM', or 'PRS'). The highest value anywhere in the namespace should
+be the :ref:`DMTF schema` package that was used to define the qualifier
+declarations and the CIM/PRS classes in that namespace.
+
+Example:
+
+.. code-block:: text
+
+    $ pywbemcli --name myserver server schema
+
+    Schema information; namespaces: all;
+    Namespace                      Schemas              classes  DMTF schema    experimental
+                                                          count  version
+    -----------------------------  -----------------  ---------  -------------  --------------
+    root                                                      0
+    root/PG_InterOp                CIM, PG                  276  2.40.0
+    root/benchmark                 CIM, (no-schema)         206  2.40.0
+    root/cimv2                     CIM, PG, PRS            1463  2.41.0         Experimental
+
+
+The above report would indicate that the namespace root/cimv2 was probably
+created with DMTF released schema version 2.41.0. Other namespaces show a
+lower level of version because they are not using any of the classes that
+have the qualifier Version value of 2.41.0.
+
+or a more detailed report (``--detail`` option):
+
+.. code-block:: text
+
+    $ pywbemcli --name myserver server schema --detail
+
+    Namespace                      Schemas              classes  schema     experimental
+                                                          count  version
+    -----------------------------  -----------------  ---------  ---------  --------------
+    root/PG_InterOp                CIM                      241  2.40.0
+                                   PG                        35  2.12.0
+    root/benchmark                 CIM                      177  2.40.0
+                                   (no-schema)               29  1.0.0
+    root/cimv2                     CIM                     1382  2.41.0     Experimental
+                                   PG                        20  2.12.0
+
+This report tells a bit more in that it reports that there are classes in the
+root/cimv2 namespace 'CIM' schema that are experimental indicating that the
+DMTF schema used was probably the Version 2.41.0, experimental DMTF schema.
```

Comment:
I suggest to remove "probably". Even if the classes are a subset of the complete CIM schema, the highest valuf of  a Version qualifier is definitely the CIM schema version, as long as the classes come consistently from a single CIM schema release.

Reply:
DONE